vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -11170,6 +11170,16 @@ quantitykind:MassPerElectricCharge
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "Mass per Electric Charge"@en ;
 .
+quantitykind:MassPerEnergy
+  a qudt:QuantityKind ;
+  qudt:hasDimensionVector qkdv:A0E-1L-2I0M1H0T2D0 ;
+  qudt:plainTextDescription "Mass per Energy is a physical quantity that can be used to relate the energy of a system to its mass." ;
+  dcterms:description "Mass per Energy (\(m/E\)) is a physical quantity that bridges mass and energy. The SI unit is \(kg/J\) or equivalently \(s^2/m^2\)."^^qudt:LatexString ;
+  # qudt:applicableUnit unit:KiloGM-PER-J ; # unit doesn't exist
```

Comment:
I'm in the process of defining it. I'm having difficulties determining the quantity kind for `KiloGM-PER-J`, as I have found that its not common to represent a unit of mass per unit energy. Do you have any recommendations?

Reply:
You are basically done for defining the quantity kind. The quantity kind is quantitykind:MassPerEnergy. The definition of unit:KiloGM-PER-J will be similar to unit:KiloGM-PER-MegaBTU_IT, but with a different conversion multiplier and different applicable systems, etc.

Reply:
Also, in case you aren't aware, all the qudt:applicableUnit triples are auto-populated as described [here](https://github.com/qudt/qudt-public-repo/wiki/Advanced-User-Guide#4-computing-applicable-units-for-a-quantitykind), so don't bother stating them in the quantity kind file - they will be overwritten anyway in the next Release.
If you need unit:KiloGM-PER-J, you will have to define that (in the units file) and be sure to include the relation qudt:hasQuantityKind quantitykind:MassPerEnergy. That way, in the next Release, the applicable unit relation pointing back from the quantity kind to the unit will be auto-populated.

Reply:
Thanks @steveraysteveray, I pushed my changes for `unit:KiloGM-PER-J`